app/javascript/chat/chat.jsx

```diff
@@ -601,6 +612,23 @@ export default class Chat extends Component {
         expanded: window.innerWidth > 600,
       });
     }
+    if (messageIsEmpty) {
+      e.preventDefault();
+
+      const messagesByCurrentUser = messages[activeChannelId].filter(
+        (message) => message.user_id === currentUserId,
+      );
+      const lastMessage =
```

Comment:
Is it possible that there are no array items at this point and `lastMessage` becomes undefined because `messagesByCurrentUser[messagesByCurrentUser.length - 1]` would be `messagesByCurrentUser[-1]`?